src/contexts/LoginContext.tsx

```diff
@@ -0,0 +1,23 @@
+import React, { createContext, useState } from "react";
+
+export interface ContextInterface{
```

Comment:
could we use a better naming to signify this is a login's context? It's better to use the understandable variable name which signifies the context.

Reply:
LoginContextInterface may be ?